bundles/org.openhab.binding.sonyprojector/src/main/java/org/openhab/binding/sonyprojector/internal/communication/SonyProjectorItem.java

```diff
@@ -12,63 +12,81 @@
  */
 package org.openhab.binding.sonyprojector.internal.communication;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.sonyprojector.internal.SonyProjectorException;
+import org.openhab.core.types.CommandOption;
 
 /**
  * Represents the different kinds of commands
  *
  * @author Markus Wehrle - Initial contribution
  * @author Laurent Garnier - Transform into an enum and rename it
+ * @author Laurent Garnier - Add more IR commands
  */
 @NonNullByDefault
 public enum SonyProjectorItem {
```

Comment:
Mostly the same capitalization comments as in the readme (e.g. "Power on/Off" should be "Power On/Off")